tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -310,10 +317,18 @@ def fit(train_data,
             where `L` ranges from 0 to 50 (Note: higher values of `L` correspond to fewer print statements, opposite of verbosity levels)
 
         Kwargs can include additional arguments for advanced users:
+            AG_args : dict, default={}
+                Keyword arguments to pass to all models. See the `AG_args` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
+                Identical to specifying `AG_args` parameter for all models in `hyperparameters`.
+                If a key in `AG_args` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
             AG_args_fit : dict, default={}
                 Keyword arguments to pass to all models. See the `AG_args_fit` argument from "Advanced functionality: Custom AutoGluon model arguments" in the `hyperparameters` argument documentation for valid values.
                 Identical to specifying `AG_args_fit` parameter for all models in `hyperparameters`.
                 If a key in `AG_args_fit` is already specified for a model in `hyperparameters`, it will not be altered through this argument.
+            AG_args_ensemble : dict, default={}
```

Comment:
Docs should touch on how is this different from `AG_args`.

Reply:
It is the same from a users POV, however AG_args_ensemble could actually pass arbitrary hyperparameters to stackers (such as if someone made a custom stacker implementation)